hudi-client/hudi-java-client/src/test/java/org/apache/hudi/client/TestJavaHoodieBackedMetadata.java

```diff
@@ -395,25 +397,27 @@ public void testMetadataArchivalCleanConfig(HoodieTableType tableType) throws Ex
         .build();
     initWriteConfigAndMetatableWriter(writeConfig, true);
 
-    AtomicInteger commitTime = new AtomicInteger(1);
     // Trigger 4 regular writes in data table.
+    List<String> instants = new ArrayList<>();
     for (int i = 1; i <= 4; i++) {
-      doWriteOperation(testTable, "000000" + (commitTime.getAndIncrement()), INSERT);
+      String instant = metaClient.createNewInstantTime();
```

Comment:
same here, maybe we can declar a variable numWrites = 4. Test is more readable and easy to understand.